.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
+  "printWidth": 100
```

Comment:
Should we add `"arrowParens": true` as recommended by the Airbnb style guide?

Reply:
Added